### PR TITLE
Improvements to the Visual Studio Project generator

### PR DESCRIPTION
**Pull request by vinniefalco**
This lets the classic (e.g. non-unity) build targets show up in the Targets dropdown and Configuration Manager.

### CODE REVIEW

**Comment by seelabs** [comment]
Instead of debug/debug.classic how about debug/debug.nounity?

**Comment by seelabs** [comment]
:+1: Targets show up correctly in drop down/all build to correct dir/regenerating from scons worked correctly.

File: SConstruct [hunk 1]
```diff
@@ -731,9 +731,14 @@ for tu_style in ['classic', 'unity']:
                 if toolchain == 'msvc':
                     config = env.VSProjectConfig(variant, 'x64', target, env)
                     msvc_configs.append(config)
+                    pass
```
Comment (seelabs) [inline SConstruct]: I don't understand why this pass is needed.

Comment (vinniefalco) [inline SConstruct]: It was leftover from commenting out the previous lines

**Comment by rec** [comment]
Reviewers: @seelabs @ximinez

**Comment by ximinez** [comment]
https://github.com/ximinez/rippled/commit/800e51d961c1a6d71df1c172f41d2021fbf250b4 updates the .sln file to reflect the new build options. I also re-ran `scons vcxproj` and included those changes. I'd strongly suggest cherry-picking this onto the branch so we don't get developer confusion.

**Comment by ximinez** [comment]
(Accidental close. Hit the wrong button.)

**Comment by vinniefalco** [comment]
@ximinez I already incorporated your changes yesterday, can you please confirm it?

**Comment by ximinez** [comment]
Weird. I must have had a cached version of this page.

:+1:

**Pull request closed without merging**